src/core/main.c

```diff
@@ -273,6 +273,14 @@ _noreturn_ static void crash(int sig) {
                         siginfo_t status;
                         int r;
 
+                        if (siginfo) {
+                                _cleanup_free_ char *cmdline = NULL;
+                                pid_t sender_pid;
+                                sender_pid = siginfo->si_pid;
```

Comment:
this can be merged in the declaration above - also add an empty new line after the declaration block